subiquity/models/tests/test_subiquity.py

```diff
@@ -77,7 +70,9 @@ def test_all(self):
         self.assertEqual(model_names.all(), {"a", "b", "c"})
 
 
-class TestSubiquityModel(unittest.IsolatedAsyncioTestCase):
+# Patch os.environ for system_scripts. Assumes tests are run in subiquity repo root.
+@mock.patch.dict(os.environ, {"SNAP": "."})
```

Comment:
i guess this could be pathlib.Path(__file__).parent.parent or whatever?

Reply:
Yeah that's cleaner I think. TIL `Path().parents`